src/plugins/data/public/search/expressions/esaggs.ts

```diff
@@ -293,6 +293,13 @@ export const esaggs = (): EsaggsExpressionFunctionDefinition => ({
             source: 'esaggs',
             sourceParams: {
               indexPatternId: indexPattern.id,
+              appliedTimeRange:
+                column.aggConfig.params.field?.name &&
+                input?.timeRange &&
+                args.timeFields &&
+                args.timeFields.includes(column.aggConfig.params.field?.name)
+                  ? { from: input.timeRange.from, to: input.timeRange.to }
+                  : undefined,
```

Comment:
Small nit, but there's enough going on here that it might be nice to have a unit test. That said, we don't have any `esaggs` tests yet 😱  --  so I'm fine holding off on it too as I plan to write some in the coming weeks as I move esaggs to the server

Reply:
Yeah, it's that kind of integration module which is super hard to unit-test...